### PR TITLE
Fullchord changes

### PR DESCRIPTION
**Pull request by Fullchord**
-rewrote civ button logic to prevent bugs/abuse
-removed ability to send civs to AI
-changed localization for game rules regarding Poland to make them clear
-changed the order of the game rules to put emphasis on the host tool
-you can now build aa in allies
-removed pp cost from leadership and social laws

### CODE REVIEW

File: common/ideas/r56i_laws_social.txt [hunk 1]
```diff
@@ -151,12 +147,11 @@ ideas = {
 			}
 			
 			modifier = {
-				political_power_factor = -0.05
 				conscription = 0.01
 				production_speed_bunker_factor = 0.15
 				production_speed_coastal_bunker_factor = 0.15
 				stability_factor = -0.05
-				army_defence_factor = 0.05
+				army_defence_factor = 0.05`
```
Comment (atomicplan666) [inline common/ideas/r56i_laws_social.txt:154]: nice typo

File: localisation/english/replace/r56_game_rules_l_english.yml [hunk 2]
```diff
@@ -632,14 +632,13 @@
...
 
- POL_BALANCE_SELECTION:0 "@POL Poland Balance"
- AI_POLAND:0 "AI Balance"
+ POL_BALANCE_SELECTION:0 "@POL Poland Nerf"
+ AI_POLAND:0 "Disabled"
  AI_POLAND_DESC:0 "Starts Poland off with 15 more divisions to make it slightly harder for the Majors to take"
- PLAYER_POLAND:0 "Player Balance"
- PLAYER_POLAND_DESC:0 "Starts Poland off with default division count balanced for players"
+ PLAYER_POLAND:0 "Enabled"
+ PLAYER_POLAND_DESC:0 "Starts Poland off with default division count balanced for when players are on Poland"
```
Comment (atomicplan666) [inline localisation/english/replace/r56_game_rules_l_english.yml:639]: The rule should say 
Disabled: "Poland is set to scuffed default"
Enabled: "If a PLAYER is on Poland then set to balanced mode"

Comment (atomicplan666) [reply localisation/english/replace/r56_game_rules_l_english.yml:639]: The game rule that turns Poland into "balanced mode" should not detect if there is a player or not (in my humble opinion). This should be left to the host's discretion.

**Review by atomicplan666 — CHANGES_REQUESTED** [review]
Fix the typo and the localization for the rules and then we good

**Review by atomicplan666 — APPROVED** [review]
looks good

**Review by Juddur — COMMENTED** [review]
YEs

**Review by Juddur — APPROVED** [review]

**Pull request merged**